core/src/tangram.cpp

```diff
@@ -241,6 +247,23 @@ void render() {
     FrameInfo::draw(*m_view, *m_tileManager);
 
     while (Error::hadGlError("Tangram::render()")) {}
+
+
+    bool ready = !(m_view->changedOnLastUpdate() ||
+                   m_tileManager->hasTileSetChanged() ||
+                   m_tileManager->hasLoadingTiles() ||
+                   m_labels->needUpdate());
+    if (ready) {
+        for (auto& ease : m_eases) {
+            if (!ease.finished()) { ready = false; }
+        }
+    }
+
+    m_renderDoneCallback(ready);
```

Comment:
Oh that makes sense! How simple :)

Would it be more natural to return the boolean from `update()`? `update()` is where tangram actually figures out whether the view is complete.